cmds/core/kexec/kexec_linux.go

```diff
@@ -2,6 +2,8 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
+//go:generate go run purgatories.go
```

Comment:
having this spread around 3 totally unrelated files is really confusing

Reply:
agreed. Just put it in one then?

Reply:
yeah +1, all in one place at least would be preferred

Reply:
Done.